CHANGELOG.md

```diff
@@ -2,466 +2,199 @@
-                    },
-                    "& $inputContainer": {
-                        width: `${200 / 3}%`
-                    }
-                }
-            }
-        }
-    }
-    ```
--   Remove default styling from `Menu` component in favor of styling the component via the theme
--   Remove `permanentMenuMinWidth` prop from `Menu` in favor of a new `variant` prop, which allows for more control. For instance, variant `temporary` can be used to give some pages more space
--   Remove `components.breadcrumbsContainer` prop form `Stack` in favor of a div that can be customized via the theme
--   Remove `FixedLeftRightLayout` component
--   Remove `FormPaper` component in favor of Material UI's [Card](https://v4.mui.com/components/cards/#card) component
--   Change default content spacing and header height of `MasterLayout`
--   Custom `headerComponent` of `MasterLayout` expects a component built using the `AppHeader` system (see [docs](https://comet-admin.netlify.app/?path=/story/docs-components-appheader--page))
--   Remove `hideToolbarMenuIcon` prop from `MasterLayout` as it is no longer necessary when building a custom header using the `AppHeader` system
--   Remove `<main>` HTML tag from `MasterLayout` in favor of new `MainContent` component
-    ```js
-    <MasterLayout headerComponent={AppHeader} menuComponent={AppMenu}>
-        <Toolbar />
-        <MainContent>
-        /* Main content goes here */
-        </MainContent>
-    </MasterLayout
-    ```
--   Remove `AppBar` inside `Tabs`. Restore the previous appearance by overriding the `CometAdminTabs-root` class
--   Remove `tabLabel` prop from `Tabs`. Use `label` instead
--   `RouterTabs` do not inherit Material UI's [Tabs props](https://v4.mui.com/api/tabs/#props) anymore. Use `tabsProps` prop to set `Tabs` props
--   Remove `showBreadcrumbs` prop from `Stack`. `StackBreadcrumbs` has been added for compatibility. However, it is recommended to use the new `Toolbar` system (handled by codemods)
-
-    ```js
-    <Stack topLevelTitle="Stack Nested">
-        <StackBreadcrumbs />
-        <StackSwitch>
-            <StackPage name="page1">
-                <Page1 />
-            </StackPage>
-            <StackPage name="page2">page2-2</StackPage>
-        </StackSwitch>
-    </Stack>
-    ```
-
--   Remove `showBackButton` prop from `Stack` (handled by codemods)
--   Remove alternating background color from body rows in `Table`. Restore the previous appearance by adding the following styles to `CometAdminTableBodyRow`:
-
-    ```js
-    odd: {
-        backgroundColor: neutrals[50],
-    }
-    ```
+-   Change the majority of classes used for style overrides. This may break custom styles
+-   Remove `showPicker` prop from `ColorPicker` and show the picker by default. Hide by using the `hidePicker` prop
+-   Remove `showClearButton` prop from `ColorPicker`. Re-add it by including the `ClearInputButton` component in the `endAdornment` prop
```

Comment:
A clearable prop was added again in https://github.com/vivid-planet/comet/commit/edb181bc9fdcb0f4ecae1f3cd3ab1ca5476e9e53

```suggestion
-   Rename the `showClearButton` prop to `clearable` in `ColorPicker`
```